api/envoy/config/trace/v3/dynamic_ot.proto

```diff
@@ -19,6 +19,8 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Dynamically loadable OpenTracing tracer]
 
+// [#not-implemented-hide:]
+//
```

Comment:
minor point: If I'm understanding correctly, we're keeping around this proto message for configuring OT even though we no longer have code to support it. Should we remove this proto message?

Reply:
probably yes - altho i wasnt 100% if we can just to that

Reply:
It would be great if the configuration file could keep working (with a deprecation warning maybe) even if it becomes no-op. Otherwise this will break the startup of envoys that configured the now dropped feature.